UI/frontend-plugins/decklink-captions/decklink-captions.cpp

```diff
@@ -67,11 +67,9 @@ void DecklinkCaptionsUI::on_source_currentIndexChanged(int)
 	captions->start();
 }
 
-static void caption_callback(void *param, obs_source_t *source,
+static void caption_callback(void *, obs_source_t *,
 			     const struct obs_source_cea_708 *captions)
 {
-	UNUSED_PARAMETER(param);
-	UNUSED_PARAMETER(source);
```

Comment:
If the functions is not from a class (so no declaration). You should put the parameter name as comment. Like it was done in AJA:

- https://github.com/obsproject/obs-studio/blob/8befd830a9807e35762006b9c715cdb133ab2c75/UI/frontend-plugins/aja-output-ui/aja-ui-main.cpp#L453
- Where it was discussed: https://github.com/obsproject/obs-studio/pull/6771#discussion_r925232452

Edit: The purpose is mostly to not lose parameters definition/name and their meaning.

Reply:
Fixed